cmd/lk/agent.go

```diff
@@ -378,6 +395,7 @@ func createAgent(ctx context.Context, cmd *cli.Command) error {
 	}
 
 	projectType, err := agentfs.DetectProjectType(workingDir)
+	fmt.Printf("Detected project type [%s]\n", util.Accented(string(projectType)))
```

Comment:
Let's just move this into `DetectProjectType()` so we don't lose it elsewhere

Reply:
well i didn't really want it in the deploy command's output and also its annoying to add to detectProjectType because either you add it all over that function or you need to refactor it to a single return statement